### ISSUE TITLE
The screen reader says nothing when navigate inside people picker suggestion list

### ISSUE THREAD

**Issue by lijunle** [OfficeDev/office-ui-fabric-react#8636]
## Codepen demonstrating the issue

Use the official web site can repro the issue, no need the codepen: https://developer.microsoft.com/en-us/fabric/#/components/peoplepicker

Steps:

1. Open the above page.
2. Launch Narrator.
3. Focus on the people picker, there will be a suggestion list pop up
   
![image](https://user-images.githubusercontent.com/1296500/55716840-8e0f3b80-5a2a-11e9-999c-7d1089dfef17.png)
4. Use up/down arrow key to navigate the suggestion list.

## Actual behavior:

- There is nothing from the screen reader during navigation.

## Expected behavior:

- The screen reader says about each suggestion item during navigation.

## Documentation describing expected behavior

Not needed.

## One more thing

P.S, during up/down navigation, each persona photo is flashing.

![PeoplePicker-PhotoFlashing](https://user-images.githubusercontent.com/1296500/55717145-46d57a80-5a2b-11e9-9d66-4ed7e66daa86.gif)

**Comment by jdhuntington** [comment]
@lijunle Thanks for reporting!

**Comment by jdhuntington** [comment]
#6259

**Comment by lijunle** [comment]
@jdhuntington I saw that page but don't find out which one is pointing to this issue. I suppose this is a regression.

**Comment by jdhuntington** [comment]
@lijunle - sorry, should have clarified. The linked issue is just a cross-reference to other accessibility work that's ongoing for this control.

**Comment by joschect** [comment]
@lijunle This is definitely a bug. I'll look into it. As far as I can tell, at least according to the [aria specs] (https://www.w3.org/TR/wai-aria-practices/#combobox), everything is correct and narrator should read it out. My best idea currently is that it might be a timing issue of when id's are put onto the page.
